network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/AbstractTopology.java

```diff
@@ -58,8 +58,11 @@ private void count(Vertex vertex) {
                     branchCount++;
                     feederCount++;
                 }
-                case LOAD, GENERATOR, BATTERY, SHUNT_COMPENSATOR, STATIC_VAR_COMPENSATOR, GROUND -> feederCount++;
+                case LOAD, GENERATOR, BATTERY, SHUNT_COMPENSATOR, STATIC_VAR_COMPENSATOR -> feederCount++;
                 case BUSBAR_SECTION -> busbarSectionCount++;
+                case GROUND -> {
```

Comment:
I don't understand why GROUND type is handled separately from LOAD, GENERATOR ...
(I saw it's the same in powsybl-core ...)

Reply:
I don't know either. @etiennehomer and I decided that it was more careful to do what is done in powsybl core but I forgot to ask why to @geofjamg